dbt_project.yml

```diff
@@ -335,9 +335,39 @@ models:
     addresses_events:
       +schema: addresses_events
       +materialized: table
+      arbitrum:
+        +schema: addresses_events_arbitrum
+        +materialized: table
+      avalanche_c:
+        +schema: addresses_events_avalanche_c
+        +materialized: table
+      bnb:
+        +schema: addresses_events_bnb
+        +materialized: table
       ethereum:
         +schema: addresses_events_ethereum
         +materialized: table
+      fantom:
+        +schema: addresses_events_fantom
+        +materialized: table
+      gnosis:
+        +schema: addresses_events_gnosis
+        +materialized: 
```

Comment:
@hildobby from the message it looks like this is causing the compile error

Reply:
one question too ser, would fantom base tables not being in the base sources file cause any issue?

Reply:
yeah @henrystats, I issued a separate PR to get the base tables addes to base sources for both Fantom and Goerli: https://github.com/duneanalytics/spellbook/pull/2342

Reply:
oh sweet awesome, I submitted a pr to add fantom base sources too but I'll close that now as I didn't include Goerli! Thank you very much ser!